Contrib/CCCS/PreCalculus/3.2/RRCC_CCCS_Openstax_AlgTrig_AT-1-001-AS_3_2_43.pg

```diff
@@ -33,41 +39,49 @@ $x3 = random(3,5,1);
 $y3 = random(-5,5,1);
 } until (abs($y1-$y2)<4 && $y1!=$y2 && ($y3- $y2)/($x3 - $x2)<2 && ($y3- $y2)/($x3 - $x2)!=($y2- $y1)/($x2 - $x1));
 
+$slope1 = ($y2- $y1)/($x2 - $x1);
+$slope2 = ($y3- $y2)/($x3 - $x2);
+
 #TEXT($BEGIN_ONE_COLUMN);
 TEXT(beginproblem());
 $showPartialCorrectAnswers = 1;
 
 $graph = init_graph(-6,-6,6,6,grid =>[12,12], axes => [0,0]);
-$f1 = new Fun( plot_list([$x1, $x2, $x3],[ $y1,$y2,$y3]),  $graph);
+#$f1 = new Fun( plot_list([$x1, $x2, $x3],[ $y1,$y2,$y3]),  $graph);
+add_functions($graph, "$slope1*(x-$x1)+$y1 for x in <$x1,$x2>". 
+  " using color:blue and weight:3");
+add_functions($graph, "$slope2*(x-$x2)+$y2 for x in <$x2,$x3>". 
+  " using color:blue and weight:3");
 $graph->stamps(open_circle($x1,$y1,'blue'), open_circle($x2,$y2,'blue'), open_circle($x3, $y3,'blue')  );
+$in=time();
+$graph->gifName($graph->gifName()."$in");
```

Comment:
This is going to cause each student to generate a new image file with every attempt, creating unnecessary bloat in the course's `tmp/` folder. 

What is this aiming to accomplish? (There's probably a better way...)